app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,7 @@
 @import 'admins';
 @import 'home';
 @import 'about';
+
+li.dropdown:hover > .dropdown-menu {
+    display: block;
```

Comment:
Indentation: 2 spaces

Reply:
Actually kalo bisa jangan kaya gini, put your own class aja di the relevant tag trus masukin display block nya ksana